src/components/ProjectFormCreate/index.js

```diff
@@ -162,6 +160,15 @@ export const ALL_USERS_QUERY = gql`
       id
       name
       email
+      avatar
```

Comment:
From line 163 ~ 171, these fields are not directly used but we need to query them for assign members feature.
Otherwise, we can't use `readFragement()` or `writeFragment()` when we try to assign members who belong to no team yet.